src/Interpreters/executeQuery.cpp

```diff
@@ -1190,6 +1201,21 @@ BlockIO executeQuery(
         if (format_name == "Null")
             streams.null_format = true;
     }
+    if (async_insert)
+    {
+        auto * queue = context->getAsynchronousInsertQueue();
```

Comment:
Missing comments in `Context.h`, before this line:
```
AsynchronousInsertQueue * getAsynchronousInsertQueue() const;
```

The previous author was wrong, but now we have to fix their faults.